front/src/app/models/scan.ts

```diff
@@ -1,8 +1,8 @@
 export interface Scan {
     id: number;
-    tool: string;
-    date: Date;
-    project: string;
+    toolName: string;
+    stringDate: string;
```

Comment:
Jeśli nie chcecie trzymać daty jako obiekt klasy Date, to lepiej jest trzymać jako liczbę, bo nie będziecie mieli problemów z ewentualnym parsowaniem stringu na datę albo pokazywaniem daty w tylko jednym konkretnym formacie.